feat: add `pre-install` script execution (#19)
laminas/laminas-continuous-integration-action: README.md

Some packages may require additional setup steps: setting up a web server to test an HTTP client, seeding a database or cache service, etc.
Other times, you may want to do additional reporting, particularly if the QA command failed.

To enable this, you may create one or more of the following files in your package:

- `.laminas-ci/pre-install.sh`
- `.laminas-ci/pre-run.sh` 
- `.laminas-ci/post-run.sh`

(Note: the files MUST be executable to be consumed!)

> These runs immediately before any other command is being executed in this action.
> The `.laminas-ci/pre-install.sh` command will receive the following arguments:

Reviewer: ```suggestion
The `.laminas-ci/pre-install.sh` command runs before any other command is executed in the action, and will receive the following arguments:
```

- `$1`: the user the QA command will run under
- `$2`: the WORKDIR path
- `$3`: the `$JOB` passed to the entrypoint (see above)

> These run immediately before and after the QA command, respectively.
> The `.laminas-ci/pre-run.sh` command will receive the following arguments:

Reviewer: ```suggestion
The `.laminas-ci/pre-run.sh` command runs immediately prior to the QA command, and will receive the following arguments:
```

You should likely audit the information on the `post-run.sh` command as well, to make sure the narrative flows correctly following these changes.

Some packages may require additional setup steps: setting up a web server to test an HTTP client, seeding a database or cache service, etc.
Other times, you may want to do additional reporting, particularly if the QA command failed.

To enable this, you may create one or more of the following files in your package:

- `.laminas-ci/pre-install.sh`
- `.laminas-ci/pre-run.sh` 
- `.laminas-ci/post-run.sh`

(Note: the files MUST be executable to be consumed!)

> The `.laminas-ci/pre-install.sh` command runs before any other command is executed in the action, and will receive the following arguments:

Reviewer: You lost the list of arguments...
